src/data_source/storage/sql/queries/node.rs

```diff
@@ -117,7 +117,7 @@ where
                 (SELECT count(*) AS null_payloads FROM payload WHERE data IS NULL) AS p,
                 (SELECT count(*) AS total_vid FROM vid) AS v,
                 (SELECT count(*) AS null_vid FROM vid WHERE share IS NULL) AS vn,
-                coalesce((SELECT last_height FROM pruned_height ORDER BY id DESC LIMIT 1)) as pruned_height
+                (SELECT(SELECT last_height FROM pruned_height ORDER BY id DESC LIMIT 1) as pruned_height)
```

Comment:
What is happening here?

Reply:
syntax error with sqlite here so had to replace the coalesce here.

Reply:
What is the double select doing?

Reply:
the outer select will return NULL if there are no rows returned by the inner query

Reply:
Ah ok